packages/desktop-client/src/components/util/GenericInput.jsx

```diff
@@ -209,15 +212,30 @@ export function GenericInput({
           />
         );
       } else {
-        content = (
-          <Input
-            inputRef={inputRef}
-            defaultValue={value || ''}
-            placeholder="nothing"
-            onEnter={e => onChange(e.target.value)}
-            onBlur={e => onChange(e.target.value)}
-          />
-        );
+        switch (type) {
+          case 'number':
+            content = (
+              <AmountInput
+                inputRef={inputRef}
+                value={amountToInteger(value)}
+                placeholder="0.00"
+                onEnter={e => onChange(e.target.value)}
+                onBlur={e => onChange(e.target.value)}
```

Comment:
You can just use onUpdate here which fires on blur

Reply:
Hi @joel-jeremy,

It's not about the signs. 
It's about setting up a split as a rule. For example, I take a total of €10 and divide it into 2 splits of €5 each.

However, as soon as I enter the value separated by a comma (German notation), the value is not saved, but 0,00 instead.